service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_3.yaml

```diff
@@ -0,0 +1,1517 @@
+                          "array_type": {
+                            "type": "primitive",
+                            "primitive_type": "File"
+                          },
+                          "non_empty": false
+                        },
+                        "record_attribute": "target_workflow_3_output_array_of_files_1"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_5",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_5"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_9",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_9"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_int_1",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "record_attribute": "target_workflow_3_output_int_1"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_3",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_3"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_2",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_2"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_12",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_12"
+                      },
+                      {
+                        "output_name": "target_workflow_3.output_file_6",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_3_output_file_6"
+                        }
```

Comment:
Do we need to add a column here for the `record_type`?

Reply:
Oops I missed that column. Will add it now 👍